### ISSUE TITLE
apple-dawin specific rust request

### ISSUE THREAD

**Issue by HughP** [Linuxbrew/homebrew-core#5501]
# Please fill out the issue checklist below, and provide all of the requested information.

# Please always follow these steps:
- [x] Confirmed this is a problem with `brew install`ing one, specific formula and not every time you run `brew`? If it's a general `brew` problem please file this issue at https://github.com/Linuxbrew/brew/issues/new.
- [ ] If you have a macOS system and can replicate the problem on macOS with Homebrew, please report the issue to [Homebrew/core](https://github.com/Homebrew/homebrew-core) and follow their issue template from your macOS system. If you are unsure, please report the issue to Linuxbrew.
- [x] Ran `brew update` and retried your prior step?
- [x] Ran `brew doctor`, fixed all issues and retried your prior step?
- [ ] Ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [ ] If `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:
- What you were trying to do (and why)
I'm trying to brew geckodriver
- What happened (include command output)
```
==> Installing geckodriver dependency: rust
==> Downloading https://static.rust-lang.org/dist/rustc-1.23.0-src.tar.gz
######################################################################## 100.0%
==> ./configure --prefix=/home/greenlantern/.linuxbrew/Cellar/rust/1.23.0 --release-channel=stable
==> make
==> make install
==> Downloading https://static.rust-lang.org/dist/2017-11-12/cargo-0.23.0-x86_64-apple-darwin.tar.gz

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "rust--cargobootstrap"
Download failed: https://static.rust-lang.org/dist/2017-11-12/cargo-0.23.0-x86_64-apple-darwin.tar.gz
```

- What you expected to happen
I would expect that the formula for rust would not be apple-darwin specific... is there a way to write this code better up stream?
- Step-by-step reproduction instructions (by running `brew install` commands)

This error should be replicable with `brew install rust`

# Formula additions or changes
To get formulae added or changed in Linuxbrew please file a [Pull Request](https://github.com/Linuxbrew/homebrew-core/blob/master/CONTRIBUTING.md).
To get formulae added or changed that is a dependency of a formula in Linuxbrew/homebrew-core submit a Pull Request to https://github.com/Linuxbrew/homebrew-core/compare
To get formulae added or changed that is not a dependency of a formula in Linuxbrew/homebrew-core submit a Pull Request to https://github.com/Linuxbrew/homebrew-extra/compare
We will close issues requesting formulae changes.

**Comment by iMichka** [comment]
I have a pull request open for that: #5410. I hope I will have time to build the bottle this weekend.

**Comment by jb2cool** [comment]
I'm having issues with rust too when installing via LinuxBrew on Ubuntu 16.04.3 and CentOS 7.4

I'm seeing;

curl: (22) The requested URL returned error: 404 Not Found
 Error: Failed to download resource "rust--cargobootstrap"
 Download failed: https://static.rust-lang.org/dist/2017-11-12/cargo-0.23.0-x86_64-apple-darwin.tar.gz

The Rust website is down
https://static.rust-lang.org/

I've logged this on their IRC channel and their forums in the first instance.

**Comment by jb2cool** [comment]
This link (https://static.rust-lang.org/dist/cargo-0.23.0-x86_64-apple-darwin.tar.gz) works so could/should the formula just be updated for that?

**Comment by iMichka** [comment]
Fix has been merged (#5410), please update.
